apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -91,27 +92,21 @@ function _isPageActive(page: INavPage): boolean {
   }
   _urlResolver.href = page.url || '';
   const target: string = _urlResolver.href;
+  let path = location.href;
 
   if (location.protocol + '//' + location.host + location.pathname === target) {
     return true;
   }
 
-  if (location.href === target) {
+  if (path === target) {
     return true;
   }
 
-  if (location.hash) {
-    // Match the hash to the url.
-    if (location.hash === page.url) {
-      return true;
-    }
-
-    // Match a rebased url. (e.g. #foo becomes http://hostname/foo)
-    _urlResolver.href = location.hash.substring(1);
-
-    return _urlResolver.href === target;
+  path = getPathMinusLastHash(path);
+  if (path === target) {
```

Comment:
Maybe I'm misunderstanding the code here--but why are we evaluating `if (path === target)` twice: once after getting the href (new line 101), then again after stripping out the hash (line 106)? Can these be simplified?

Reply:
Very fair point. It's in there twice because of the double hash situation - we only need the second `if (path === target)` for those cases. I refactored to do a hash count with one  `if (path === target)`  to make it more clear what the code is aiming for.